app/create-event/page.tsx

```diff
@@ -75,16 +78,7 @@ const CreateEvent: React.FC = () => {
     const { isAuth, user } = useAuth();
 
     if (!isAuth || (user && user.role !== 'admin' && user.role !== 'creator')) {
-      return (
-        <Box sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center', height: '80vh' }}>
-          <Typography variant="h5" component="h2">
-            You are not authorized to access this page.
-          </Typography>
-        </Box>
-      );
-    }
-  
-  
+
```

Comment:
Hi Kai, can you explain your reasoning for leaving this condition statement empty?